stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -134,6 +152,14 @@ public static Token retrieveToken(
         for (String propertyName : propertyNames) {
             propertyMap.put(propertyName, System.getProperty(propertyName));
         }
+
+        propertyMap.put("phone.brand", Build.BRAND);
```

Comment:
Hey! Actually, none of this stuff will actually make it into any meaningful data store unless it's already part of our pre-defined user agent schema. Even before your change here, all those debug headers were already being ignored as far as I can tell.

The canonical keys are: `uname`, `bindings_version`, `lang`, `lang_version`.